### DISCUSSION TITLE
‎GitHub Flavored Markdown (GFM) and links inside a table cell

### DISCUSSION THREAD

**Discussion by 4r73m190r0s** [community/community#57810] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

[GitHub repo example](https://github.com/4r73m190r0s/markdown-test/blob/main/README.md)

I'm using [this](https://www.markdownguide.org/extended-syntax/#linking-to-heading-ids) syntax to create custom links to header, but it doesn't work inside table cells, but it does work outside of it (as can be seen in the example above).

I've seen some proposals to add spaces between HTML tags and Markdown syntax, but it didn't help.

**Comment by Gustavonn07** [comment, marked as answer]
I didn't quite understand the question, but if the problem is leaving the title formatted inside the table:

Instead of using # Title use <.h1> Title <./h1>
 (dont use .  I just used it to be visible)

Inside tables it is better to use tags to work

**Reply by 4r73m190r0s** [reply]
The problem was that markdown syntax for linking doesn't work inside table cells. 

Your solution works. I've defined header with html tags and `id` attribute, and later used html `<a>` tag for links, instead of markdown `[Link](url)` syntax.
